README.md

```diff
@@ -85,6 +85,59 @@ git fetch upstream-code main
+Also the file serves data mapping for provided branding resources.
+
+Example of the `branding/product.json` file:
+```
+{
+	"nameShort": "Visual Studio Code - Open Source",
+	"nameLong": "Red Hat OpenShift Dev Spaces - Visual Studio Code - Open Source IDE container",
+	"icons": {
+		"favicon": {
+			"universal": "icons/favicon.ico"
+		},
+		"welcome": {
+			"universal": "icons/dev-spaces.svg"
+		},
+		"statusBarItem": {
+			"universal": "icons/dev-spaces.svg"
+		},
+		"letterpress": {
+			"light": "icons/letterpress-light.svg",
+			"dark": "icons/letterpress-light.svg"
+		}
+	},
+	"remoteIndicatorCommands": {
+		"openDocumentationCommand": "Dev Spaces: Open Documentation",
+		"openDashboardCommand": "Dev Spaces: Open Dashboard",
+		"stopWorkspaceCommand": "Dev Spaces: Stop Workspace"
+	},
+	"workbenchConfigFilePath": "workbench-config.json",
+	"codiconCssFilePath": "css/codicon.css"
+}
+```
+- `nameShort` - application name
+- `nameLong` - it's used for the `Welcome` page, the `About` dialog, browser tab title
+- `favicon` - icon for the browser tab title, the same for all themes
+- `welcome` - icon for the `Welcome` (`Get Started`) page tab title, the same for all themes
+- `statusBarItem` - icon for the status bar item, the same for all themes, should be defined as `codicon` in the `workbench-config.json` file and `codicon` css styles as well
+- `letterpress` - icon for the main area when all editors are closed, it's possible to provide different icons for the `light` and `dark` theme  
+- `remoteIndicatorCommands` - names of commands provided by [`Eclipse Che Remote` extension](https://github.com/che-incubator/che-code/blob/main/code/extensions/che-remote/package.nls.json)
+- `workbenchConfigFilePath` - the config file path, please see an example of the config file [here](https://github.com/che-incubator/che-code/blob/main/code/src/vs/code/browser/workbench/che/workbench-config.json)
+- `codiconCssFilePath` - the codicon css file path, should contain css styles for `codicon`s, content of the file is appended to [the coressponding css file](https://github.com/che-incubator/che-code/blob/main/code/src/vs/base/browser/ui/codicons/codicon/codicon.css), please see an example of the content [here](https://github.com/che-incubator/che-code/blob/803b864e8411bd57d617dabddfd8a132fac6c743/code/src/vs/base/browser/ui/codicons/codicon/codicon.css#L29-L33)
+
+Please note:
+-  it's expected that all paths in the `branding/product.json` file are relative to the `branding` folder. 
+For example, `workbenchConfigFilePath` field has `anyFolder/myConfigFiles/workbench-config.json` value. 
+It means that the config file can be found by the path: `che-code/branding/anyFolder/myConfigFiles/workbench-config.json`
+- At the moment the [branding.sh script](https://github.com/che-incubator/che-code/blob/main/branding/branding.sh) is not run automatically (at building project, for example), it should be integrated into the build process of the project.
```

Comment:
```suggestion
- At the moment the [branding.sh script](https://github.com/che-incubator/che-code/blob/main/branding/branding.sh) is not run automatically when building this project. It needs to be integrated into the build process of the [downstream branded project or product](https://github.com/redhat-developer/devspaces-images/blob/devspaces-3-rhel-8/devspaces-code/build/scripts/sync.sh#L96).
```

Reply:
> In the future, this script will be integrated

If we speak about `devspaces-code` - it's already integrated, please see [here](https://github.com/redhat-developer/devspaces-images/blob/devspaces-3-rhel-8/devspaces-code/build/scripts/sync.sh#L96)

In my sentence above I meant that the script should be integrated for other projects based on the `che-code`